createManifest.ps1

```diff
@@ -26,14 +26,21 @@ param(
     $TagList = ('ubuntu-16.04', 'windowsservercore')
 )
 
+$first = $true
 $manifestList = @()
 foreach($tag in $TagList)
 {
-    $manifestList += "$ContainerRegistry/${Image}:$tag"
+    $ammend = ""
+    if (!$first) {
+        $amend = '--amend'
+    }
+
+    Write-Verbose -Message "running: docker manifest create $ammend $ContainerRegistry/${Image}:$ManifestTag $ContainerRegistry/${Image}:$tag" -Verbose
+    docker manifest create $amend $ContainerRegistry/${Image}:$ManifestTag "$ContainerRegistry/${Image}:$tag"
+    $first = $false
 }
 
 # Create the manifest
-docker manifest create $ContainerRegistry/${Image}:$ManifestTag $manifestList
 
```

Comment:
This is the main fix.  docker manifest create was not including all specified manifests.

switching to using --amend and building it up one by one works fine.

Reply:
There's a typo in two uses of the variable (`$ammend` instead of `$amend`), which means this change won't actually have any effect.

Reply:
@TBBle Only in the initialization to empty... I'll fix it in another, PR.  Thanks.

Reply:
Created https://github.com/PowerShell/PowerShell-Docker/pull/468/files